c7n/resources/iam.py

```diff
@@ -1119,6 +1119,91 @@ def process(self, resources, event=None):
         return []
 
 
+# ############### begin  CIEM Role filter ##################
+
+@Role.filter_registry.register('has-inline-policy-attached')
+class IamRoleInlinePolicy(Filter):
+    """
+        Filter IAM roles that have an inline-policy attached
+
```

Comment:
Please fix the variable name:

https://stackoverflow.com/questions/159720/what-is-the-naming-convention-in-python-for-variable-and-function